packages/components/src/VirtualizedList/RowCollapsiblePanel/RowCollapsiblePanel.component.js

```diff
@@ -0,0 +1,90 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import classNames from 'classnames';
+import { CellMeasurer } from 'react-virtualized';
+import CollapsiblePanel from '../../CollapsiblePanel/CollapsiblePanel.component';
+import { extractSpecialFields, getCellData, getId, getRowData } from '../utils/gridrow';
+
+import withListGesture from '../../Gesture/withListGesture';
+import css from './RowCollapsiblePanel.scss';
+
+const DEFAULT_HEIGHT = 50;
+
+/**
+ * Row renderer that displays a Collapsible Panel
+ */
+class RowCollapsiblePanel extends React.Component {
+	render() {
+		const { className, index, onKeyDown, parent, style } = this.props;
+		const { titleField } = extractSpecialFields(parent);
+
+		const parentId = getId(parent);
+		const id = parentId && `${parentId}-${index}`;
+		const rowData = getRowData(parent, index);
+
+		let onRowDoubleClick;
+
+		if (parent.props.onRowDoubleClick) {
+			onRowDoubleClick = event => parent.props.onRowDoubleClick({ event, rowData });
+		}
+
+		const onToggle = (event, measure) => {
+			if (parent.props.onRowClick) {
+				parent.props.onRowClick({ event, rowData: { ...rowData, index } });
+				setTimeout(measure, 0);
```

Comment:
I don't understand the defer here.
But that doesn't work if you already have expanded panels as initial props.
You can instead use onLoad attribute on the div to execute measure()